TrafficCapture/dockerSolution/src/main/docker/migrationConsole/workflows/templates/replayer.yaml

```diff
@@ -0,0 +1,220 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: replayer
+spec:
+  entrypoint: deploy-replayer
+  serviceAccountName: argo-workflow-executor
+  templates:
+    - name: deploy-replayer
+      inputs:
+        parameters:
+            # required
+            - name: target-url
+
+            # required, but has defaults
+            - name: image
+              value: "migrations/traffic_replayer:latest"
+            - name: replicas
+              value: "1"
+
+            # optional with defaults
+            - name: packet-timeout-seconds
+              value: "70"
+            - name: max-concurrent-requests
+              value: "1024"
+            - name: num-client-threads
+              value: "0"
+            - name: lookahead-time-window
+              value: "300"
+            - name: speedup-factor
+              value: "1.0"
+            - name: target-response-timeout
+              value: "30"
+            - name: insecure
+              value: "false"
+
+            # optional
+            - name: input-file
+              value: ""
+            - name: auth-header-value
+              value: ""
+            - name: auth-header-user-and-secret
+              value: ""
+            - name: remove-auth-header
+              value: ""
+            - name: sigv4-auth-header-service-region
+              value: ""
+            - name: user-agent
+              value: ""
+            - name: transformer-config
+              value: ""
+            - name: transformer-config-encoded
+              value: ""
+            - name: transformer-config-file
+              value: ""
+            - name: tuple-transformer-config
+              value: ""
+            - name: tuple-transformer-config-base64
+              value: ""
+            - name: tuple-transformer-config-file
+              value: ""
+            - name: kafka-traffic-topic
+              value: ""
+            - name: kafka-traffic-brokers
+              value: ""
+            - name: kafka-traffic-group-id
+              value: ""
+            - name: kafka-traffic-property-file
+              value: ""
+            - name: kafka-traffic-enable-msk-auth
+              value: ""
+            - name: otel-collector-endpoint
+              value: "http://otel-collector:4317"
+
+      resource:
+        action: create
+        setOwnerReference: true
+        manifest: |
+          apiVersion: apps/v1
+          kind: Deployment
+          metadata:
+            generateName: replayer-
+            labels:
+              app: replayer
+          spec:
+            replicas: {{inputs.parameters.replicas}}
+            selector:
+              matchLabels:
+                app: replayer
+            template:
+              metadata:
+                labels:
+                  app: replayer
+              spec:
+                containers:
```

Comment:
I'd like to see us make this pattern for argument passing a bit cleaner for proxy and replayer. Ideally we would be able to loop over the input parameters but don't know if that is supported